docs/user-guide/managing-deployments.md

```diff
@@ -230,195 +211,118 @@ For more information, please see [annotations](/docs/user-guide/annotations/) an
+```shell 
+$ kubectl autoscale deployment/my-nginx --min=1 --max=3
+deployment "my-nginx" autoscaled
+```
 
-To update a service without an outage, `kubectl` supports what is called ['rolling update'?](/docs/user-guide/kubectl/kubectl_rolling-update), which updates one pod at a time, rather than taking down the entire service at the same time. See the [rolling update design document](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/design/simple-rolling-update.md) and the [example of rolling update](/docs/user-guide/update-demo/) for more information.
+Now your nginx replicas will be scaled up and down as needed, automatically. 
 
-Let's say you were running version 1.7.9 of nginx:
+For more information, please see [kubectl scale](/docs/user-guide/kubectl/kubectl_scale/), [kubectl autoscale](/docs/user-guide/kubectl/kubectl_autoscale/) and [horizontal pod autoscaler](/docs/user-guide/horizontal-pod-autoscaler/) document.
 
-```yaml
-apiVersion: v1
-kind: ReplicationController
-metadata:
-  name: my-nginx
-spec:
-  replicas: 5
-  template:
-    metadata:
-      labels:
-        app: nginx
-    spec:
-      containers:
-      - name: nginx
-        image: nginx:1.7.9
-        ports:
-        - containerPort: 80
-```
 
-To update to version 1.9.1, you can use [`kubectl rolling-update --image`](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/design/simple-rolling-update.md):
+## In-place updates of resources
 
-```shell
-$ kubectl rolling-update my-nginx --image=nginx:1.9.1
-Created my-nginx-ccba8fbd8cc8160970f63f9a2696fc46
-```
+Sometimes it's necessary to make narrow, non-disruptive updates to resources you've created. 
 
-In another window, you can see that `kubectl` added a `deployment` label to the pods, whose value is a hash of the configuration, to distinguish the new pods from the old:
+### kubectl apply
 
-```shell
-$ kubectl get pods -lapp=nginx -Ldeployment
-NAME                                              READY     STATUS    RESTARTS   AGE       DEPLOYMENT
-my-nginx-ccba8fbd8cc8160970f63f9a2696fc46-k156z   1/1       Running   0          1m        ccba8fbd8cc8160970f63f9a2696fc46
-my-nginx-ccba8fbd8cc8160970f63f9a2696fc46-v95yh   1/1       Running   0          35s       ccba8fbd8cc8160970f63f9a2696fc46
-my-nginx-divi2                                    1/1       Running   0          2h        2d1d7a8f682934a254002b56404b813e
-my-nginx-o0ef1                                    1/1       Running   0          2h        2d1d7a8f682934a254002b56404b813e
-my-nginx-q6all                                    1/1       Running   0          8m        2d1d7a8f682934a254002b56404b813e
-```
+The most disciplined way to update resources is [`kubectl apply`](/docs/user-guide/kubectl/kubectl_apply/). If you maintain a set of configuration files in source control, 
```

Comment:
Wording could be better here, but I don't want to block on it.

Would also be useful to link to ([configuration as code](http://martinfowler.com/bliki/InfrastructureAsCode.html)).